apps/docs/src/routes/playground/docgen.ts

```diff
@@ -1,21 +1,44 @@
+import { parse, createCacheStorage } from "svelte-docgen";
+import { ssp } from "sveltekit-search-params";
 import ts from "typescript";
 import * as tsvfs from "@typescript/vfs";
 
-import { parse, createCacheStorage, encode } from "svelte-docgen";
-
+// TODO: This can be removed once we migrate away from using `svelte2tsx`
 import shim from "svelte2tsx/svelte-shims-v4.d.ts?raw";
 
-export const COMPILER_OPTIONS: ts.CompilerOptions = {
+export async function generate_docgen(source: string) {
+	const fsmap = await tsvfs.createDefaultMapFromCDN(COMPILER_OPTIONS, ts.version, false, ts, {
```

Comment:
This should also be called only once and reused, even though it has some caching mechanism.